gossip/resolver/resolver_test.go

```diff
@@ -34,14 +34,14 @@ func TestParseResolverSpec(t *testing.T) {
 		resolverAddress string
 	}{
 		// Ports are not checked at parsing time. They are at GetAddress time though.
-		{"127.0.0.1:26257", true, "tcp", "127.0.0.1:26257"},
-		{":26257", true, "tcp", util.EnsureHost(":26257")},
-		{"127.0.0.1", true, "tcp", "127.0.0.1"},
-		{"tcp=127.0.0.1", true, "tcp", "127.0.0.1"},
-		{"lb=127.0.0.1", true, "lb", "127.0.0.1"},
+		{"127.0.0.1:26222", true, "tcp", "127.0.0.1:26222"},
```

Comment:
you mean test case like:
{"tcp=127.0.0.1:23456", true, "tcp", "127.0.0.1:23456"},